terraform/modules/networking/security_groups.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_default_security_group" "main" {
+  vpc_id = aws_vpc.main.id
+}
+
+resource "aws_default_security_group" "default_eu_west_2" {
```

Comment:
Again I felt it was better to keep the default provider (eu-west-2) at the top, even though this isn't in alphanumeric order.